example/package.json

```diff
@@ -30,9 +30,9 @@
   "license": "ISC",
   "type": "module",
   "dependencies": {
-    "@verify-media/verify-client": "^1.0.10",
+    "@verify-media/verify-client": "~1.1.0",
     "ethers": "^5.7.0",
-    "siwe": "^2.0.5",
+    "siwe": "2.1.4",
```

Comment:
this works?

Reply:
anything I missed Ankit? I copied the same change from the main package json